### PR TITLE
Updates as per released FDO spec

### PR DESCRIPTION
**Pull request by prateekcr**
- Updating SHA256 and SHA384 constants.
- Updating the names of Nonces.

Signed-off-by: Chandrakar, Prateek <prateek.chandrakar@intel.com>

### CODE REVIEW

File: lib/prot/to2/msg65.c [hunk 1]
```diff
@@ -131,23 +131,23 @@ int32_t msg65(fdo_prot_t *ps)
...
-		LOG(LOG_ERROR, "TO2.SetupDevice: Failed to alloc Nonce7\n");
+	ps->nonce_to2setupdv_rcv = fdo_byte_array_alloc(FDO_NONCE_BYTES);
+	if (!ps->nonce_to2setupdv_rcv) {
+		LOG(LOG_ERROR, "TO2.SetupDevice: Failed to alloc NonceTO2SetupDv\n");
 		goto err;
 	}
-	if (!fdor_byte_string(&ps->fdor, ps->n7r->bytes, FDO_NONCE_BYTES)) {
-		LOG(LOG_ERROR, "TO2.SetupDevice: Failed to read rNonce7\n");
+	if (!fdor_byte_string(&ps->fdor, ps->nonce_to2setupdv_rcv->bytes, FDO_NONCE_BYTES)) {
+		LOG(LOG_ERROR, "TO2.SetupDevice: Failed to read rNonceTO2SetupDv\n");
```
Comment (trbehera) [inline lib/prot/to2/msg65.c:144]: rNonceTO2SetupDV => NonceTO2SetupDV

File: lib/prot/to2/msg61.c [hunk 2]
```diff
@@ -189,31 +189,33 @@ int32_t msg61(fdo_prot_t *ps)
...
+		LOG(LOG_ERROR, "TO2.ProveOVHdr: Failed to read NonceTO2ProveOV\n");
 		goto err;		
 	}
 
-	/* The nonces "n5" (msg40) and "n6" here must match */
-	if (!fdo_nonce_equal(ps->n5r, ps->n5)) {
-		LOG(LOG_ERROR, "TO2.ProveOVHdr: Received Nonce5 and Nonce5 do not match\n");
+	/* The nonces "NonceTO2ProveOV" from Type 60 and 61 must match */
+	if (!fdo_nonce_equal(ps->nonce_to2proveov_rcv, ps->nonce_to2proveov)) {
+		LOG(LOG_ERROR, "TO2.ProveOVHdr: Received NonceTO2ProveOV and NonceTO2ProveOV"
```
Comment (trbehera) [inline lib/prot/to2/msg61.c:209]: Can we update this inline with msg/67 log?

`TO2.ProveOVHdr: Received NonceTO2ProveOV does not match with existing NonceTO2ProveOV`

File: lib/include/fdoprot.h [hunk 3]
```diff
@@ -214,12 +214,12 @@ typedef struct fdo_prot_s {
 	int owner_supplied_service_info_num;
 	int owner_supplied_service_info_rcv;
 	fdo_owner_supplied_credentials_t *osc;
-	fdo_byte_array_t *n4;
-	fdo_byte_array_t *n5;
-	fdo_byte_array_t *n7;
-	fdo_byte_array_t *n5r;
-	fdo_byte_array_t *n6;
-	fdo_byte_array_t *n7r;
+	fdo_byte_array_t *nonce_to1proof;
```
Comment (trbehera) [inline lib/include/fdoprot.h:217]: Consider sorting them.

Suggested:
```
fdo_byte_array_t *nonce_to1proof;
fdo_byte_array_t *nonce_to2proveov;
fdo_byte_array_t *nonce_to2proveov_rcv;
fdo_byte_array_t *nonce_to2provedv;
fdo_byte_array_t *nonce_to2setupdv;
fdo_byte_array_t *nonce_to2setupdv_rcv;
```

**Comment by prateekcr** [comment]
Updated as per review comments.

**Review by trbehera — APPROVED** [review]

**Review by johneasterday — APPROVED** [review]

**Pull request merged**